src/components/input/Input.tsx

```diff
@@ -24,6 +25,8 @@ export const Input = React.forwardRef<HTMLInputElement, Props>(
     },
     ref
   ) => {
+    const savedThemeValue = window.localStorage.getItem("NC_GNUI_THEME") ?? "";
```

Comment:
I think this might not work when we will dynamically switch the theme in the app, right?
In a case when Input is already visible & won't rerender (It will need some update e.g. entered value for new styles to be updated).

Reply:
Since `useThemeSwitcher` should be module-cached, I think we could do 

```ts
const { currentTheme } = useThemeSwitcher();
```

And the value should auto-update when the theme is changed in the app.